service_client_initializer.go

```diff
@@ -0,0 +1,221 @@
+package device
```

Comment:
snake_case isn't used in the other SDK files, so maybe the file name should be changed
There is a related issue in [edgex-go](https://github.com/edgexfoundry/edgex-go/issues/535)

Reply:
I change it to clientinitializer